src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -67,12 +68,10 @@ private enum ParameterSyntaxKind : byte
             }
 
             _lazyDefaultSyntaxValue = defaultSyntaxValue;
+            ParameterBinder = parameterBinderOpt;
         }
 
-        protected virtual Binder ParameterBinder
-        {
-            get { return null; }
-        }
+        protected virtual Binder ParameterBinder { get; }
```

Comment:
>protected virtual Binder ParameterBinder { get; } [](start = 8, length = 49)

It feels like this property can be calculated from the containing LocalFunctionSymbol.